Assets/Scripts/GameData/GameData.cs

```diff
@@ -90,11 +138,31 @@ public static class GameData {
 		return stats.Find(x => x.name == nameKey);
 	}
 
+	// Called to retrieve static ability data
+	public static AbilityData Abilities(string nameKey) {
+		return abilities.Find(x => x.name == nameKey);
+	}
+
+	// Called to retrieve static action data
+	public static ActionData Actions(string nameKey) {
+		return actions.Find(x => x.name == nameKey);
+	}
+
+	// Called to retrieve static perk data
+	public static PerkData Perks(string nameKey) {
+		return perks.Find(x => x.name == nameKey);
+	}
+
```

Comment:
Are these getter functions?  If so maybe consider renaming these like getPerk, getAction, etc.  Actions(param) seems more like a constructor for a class than it does a getter function.